app/perfunwind.cpp

```diff
@@ -653,6 +656,14 @@ void PerfUnwind::analyze(const PerfRecordSample &sample)
         }
     }
 
+    // FIXME: use a command line switch
+    if (qEnvironmentVariableIsSet("STITCH_CALLCHAIN")) {
+        if (m_currentUnwind.firstGuessedFrame != -1)
```

Comment:
I would recommend this gets pushed into the stitchCurrentCallchain method (to do an early-return there), dito with the cli/env var check